msu/systems/keybinds/keybind_sq.nut

```diff
@@ -16,6 +17,12 @@
 		this.KeyState = _keyState;
 	}
 
+	function setBypassInputDenied(_bool)
+	{
+		this.BypassInputDenied = _bool;
+		return this;
```

Comment:
As much as I agree with returning this here in principle, we haven't been doing this. I think this would be a useful thing to refactor in the codebase as a whole but doing this in just this one function seems wrong to me, so it should be a separate PR/commmit